### PR TITLE
Add reading FEC file functionnality

### PR DESCRIPTION
**Pull request by arnaudgoulpeau**
- add new service to read FEC files
- updated Ecriturexxx input objects to make them in/out objects :
namespace changed to ValueObject
- updated normalizers to create objects from raw FEC data (from file)
- updated computer for the same purpose
- add new "CsvReader" service to read tab and pipe separated CSV files

### CODE REVIEW

File: Reader/CsvReader.php [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+     * Reads the CSV FEC
+     * @param File  $file
+     * @return array<array>
+     */
+    public function read(File $file)
+    {
+        $splFileObject = $file->openFile('r');
+
+        // write csv data
+        return $this->readCsv($splFileObject);
```
Comment (thomasbeaujean) [inline Reader/CsvReader.php]: I think it should be 

    $data = $this->readCsv($splFileObject);

and 

    return $data

at the end of the method

File: Reader/CsvReader.php [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+     *
+     * @throw FecException
+     * @return array the modified row
+     */
+    protected function convertDateFields(array $row, $cnt)
+    {
+        $fields = ['EcritureDate', 'PieceDate', 'DateLet', 'ValidDate', 'DateRglt'];
+        foreach ($fields as $field) {
+            if (isset($row[$field])) {
+                if ($row[$field] != "") {
```
Comment (thomasbeaujean) [inline Reader/CsvReader.php]: please use simpleQuote please and strict comparison operators (!==)

File: Service/FecReader.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace A5sys\FecBundle\Service;
+
+use A5sys\FecBundle\Computer\DebitCredit\DebitCreditComputerInterface;
+use A5sys\FecBundle\Reader\ReaderInterface;
```
Comment (thomasbeaujean) [inline Service/FecReader.php]: "use" in alpha order please

**Review by thomasbeaujean — CHANGES_REQUESTED** [review]

File: Reader/CsvReader.php (hunk 1 above)
Comment (arnaudgoulpeau) [reply Reader/CsvReader.php]: Right ! Thanks

**Pull request merged**